lib/commands/publish.js

```diff
@@ -185,7 +186,8 @@ Consider creating a release on GitHub by yourself with this message:\n${releasin
       name: `v${releasing.version}`,
       body: releasing.message,
       draft: args.ghReleaseDraft,
-      prerelease: args.ghReleasePrerelease
+      prerelease: args.ghReleasePrerelease,
+      generate_release_notes: args.ghReleaseBody
```

Comment:
When `args.ghReleaseBody` is true, I would expect that `releasing.message` is not generated. So `const releasing = await draft(args)` requires to manage the flag as well and return just `null` in this case

Reply:
Sounds good! Updated to the match above. Also removed any logic to get logs [here](https://github.com/fastify/releasify/pull/257/files#diff-1dedbe8ba47947141c90659de58ea869c28396cae2ff5099c8364f8d1d6ea8f4R56). Figured since gh will generate release notes we didn't need to retrieve them